### PR TITLE
Chroma Link Support

### PR DESCRIPTION
**Pull request by Mockarutan**
Added support for Chroma Link. Not integrated in any tests.

Implements #225.

(First time EVER I use git hub (SVN is my jam), let alone making a pull request to a open source project like this, so sorry if anything is wrong.)

### CODE REVIEW

File: Corale.Colore/Core/ChromaLink.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    using System.Linq;
+
+    using Corale.Colore.Annotations;
+    using Corale.Colore.Logging;
+
+    using Razer.ChromaLink;
+    using Razer.ChromaLink.Effects;
+
+    /// <summary>
+    /// Class for interacting with a Chroma keyboard.
```
Comment (njbmartin) [inline Corale.Colore/Core/ChromaLink.cs]: All references to keyboard should be updated

**Comment by njbmartin** [comment]
Other than the modifications to the `.sln` file and my comment about keyboard mentions, the changes look good.

File: Corale.Colore/Core/ChromaLink.cs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    /// </summary>
+    [PublicAPI]
+    public sealed class ChromaLink : Device, IChromaLink
+    {
+        /// <summary>
+        /// Logger instance for this class.
+        /// </summary>
+        private static readonly ILog Log = LogManager.GetLogger(typeof(Keypad));
```
Comment (Sharparam) [inline Corale.Colore/Core/ChromaLink.cs]: Should be `LogManager.GetLogger(typeof(ChromaLink));`.

**Comment by Sharparam** [comment]
The CI server is currently down for unknown reasons (I have yet to hear from @brandonscott who is hosting it). Until it is back up or I am back on my development machine I cannot test the changes.

**Comment by njbmartin** [comment]
Will take a look later!

**Comment by njbmartin** [comment]
Tests pass 276/276

**Comment by roxaskeyheart** [comment]
Not sure how related this is to ChromaLink yet (mine is still on the way from Razer), but just going to post this here as well:

https://www.razerzone.com/gaming-accessories/razer-chroma-hdk

Hopefully it's pretty easy to implement too. I might give it a try and submit a Pull Request when I get mine if no one else has in the mean time.

**Comment by Sharparam** [comment]
Probably going to be added as another API function. @roxaskeyheart can you do the changes I listed (adding getters)?

**Comment by roxaskeyheart** [comment]
I have done the changes and some more for the ChromaLink changes you wanted above. I don't have access to contribute to this PR though, is there anyway you can add me to it or should I submit a new PR instead?

**Comment by Sharparam** [comment]
Something must've broken when I changed the target branch on this PR. Closing this in favor of #229.

**Pull request closed without merging**